tests/integration/test_yahoo_source.py

```diff
@@ -0,0 +1,34 @@
+from datetime import datetime, timedelta
+from typing import List
+from unittest import TestCase
+
+from entities.candle import Candle
+from entities.timespan import TimeSpan
+from market.yahoofinance.history_provider import YahooFinanceHistoryProvider
+from pipeline.sources.yahoo_finance_history import YahooFinanceHistorySource
+
+
+class TestYahooMarketSource(TestCase):
+    provider = YahooFinanceHistoryProvider()
+    symbols = ['AAPL', 'MSFT']
+    from_time = datetime.now() - timedelta(days=50)
```

Comment:
A static datetime would prevent any irregularities on the test, what do you think?

Reply:
I Agree. 
Done.